<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoTab</class>
 <widget class="QWidget" name="AutoTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <widget class="QFrame" name="currentPFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="5" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="threeThirtyP">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>330'</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="quarterLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1/4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="sixtyD">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>60'</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="threeThirtyA">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>330'</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="threeThirtyLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>330'</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="quarterD">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1/4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="quarterH">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="thousandA">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1000'</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="sixtyA">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>60'</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="threeThirtyH">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>330'</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="sixtyH">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>60'</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="temperatureLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>By Temperature</string>
        </property>
       </widget>
      </item>
      <item row="5" column="6" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="quarterP">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="currentPLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Predicted Run</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="eighthH">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="eighthA">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/8</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="sixtyP">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>60'</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="thousandD">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1000'</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="quarterA">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="thousandP">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1000'</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="eighthD">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1/8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="eighthLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1/8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="sixtyLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>60'</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="threeThirtyT">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>330'</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="threeThirtyD">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>330'</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="pressureLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>By Pressure</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="humidityLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>By Humidity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="quarterT">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="averageLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>By Average Weather</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="eighthT">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/8</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="thousandH">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1000'</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="thousandT">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1000'</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="densityALabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>By Density Altitude</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="thousandLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1000'</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="eighthP">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/8</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="sixtyT">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>60'</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="7" column="0">
       <widget class="ProviderBox" name="textProviderComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="weightAdjustmentSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>0.999000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="eToPhoneCheckBox">
        <property name="text">
         <string>Eighth to Phone</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="textNumberEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="vehicleWeightSpinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="windAdjustmentSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>0.990000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QDoubleSpinBox" name="riderWeightSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="qToPhoneCheckBox">
        <property name="text">
         <string>Quarter to Phone</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="textProviderLabel">
        <property name="text">
         <string>Service Provider</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" alignment="Qt::AlignRight">
       <widget class="QLabel" name="windAdjustmentLabel">
        <property name="text">
         <string>1/4mi ET adjustment per 1MPH of wind</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2" alignment="Qt::AlignRight">
       <widget class="QLabel" name="weightAdjustmentLabel">
        <property name="text">
         <string>1/4mi ET adjustment per 1lb of weight</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2" alignment="Qt::AlignRight">
       <widget class="QLabel" name="vehicleWeightLabel">
        <property name="text">
         <string>Vehicle Weight</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2" alignment="Qt::AlignRight">
       <widget class="QLabel" name="riderWeightLabel">
        <property name="text">
         <string>Rider Weight</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="autoPredictCheckBox">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Make Automatic Predictions from Weather Readings</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="previousPredictionsButton">
        <property name="text">
         <string>Show Previous Predictions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="trackTicketsCheckBox">
        <property name="text">
         <string>Use all tickets for this track</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="vehicleTicketsCheckBox">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Use all tickets for this vehicle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QFrame" name="graphFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1/8 Mile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>1/4 Mile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProviderBox</class>
   <extends>QComboBox</extends>
   <header>providerbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
